src/baldr/laneconnectivity.cc

```diff
@@ -1,8 +1,8 @@
 #include "baldr/laneconnectivity.h"
+#include "midgard/logging.h"
 #include <boost/algorithm/string/classification.hpp>
 #include <boost/algorithm/string/split.hpp>
 #include <string.h>
-#include <valhalla/midgard/logging.h>
```

Comment:
the bulk of the changes in this pr are just fixing includes like this. ill comment only on the substantive parts of the rest of the pr